waltz-tools/src/test/java/com/wepay/waltz/tools/client/ClientCliTest.java

```diff
@@ -38,12 +39,18 @@ public void restoreStreams() {
         System.setErr(originalErr);
     }
 
-    Properties createProperties(String connectString, String znodePath, int sessionTimeout, SslSetup sslSetup) {
+    Properties createProperties(String connectString,  String znodePath, int sessionTimeout, SslSetup sslSetup) {
+        return createProperties(connectString, znodePath, sessionTimeout, sslSetup,
+            WaltzClientConfig.CLIENT_SSL_CONFIG_PREFIX);
```

Comment:
Nit: not sure if this method is needed. Can't we keep only the generic one below and fill "WaltzClientConfig.CLIENT_SSL_CONFIG_PREFIX" wherever needed ?

Reply:
ohh, I just thought this would be neat compared to adding `WaltzClientConfig.CLIENT_SSL_CONFIG_PREFIX` argument to all createProperties(..) method call.
I can update the code to use the generic createProperties method itself.